www/apps/docs/content/upgrade-guides/medusa-ui/2-0-0.md

```diff
@@ -0,0 +1,71 @@
+---
+description: "Actions Required for v.2.0.0"
+---
+
+# v2.0.0
+
+Version 2.0.0 of Medusa UI introduces a set of new components, and breaking changes to some of the existing components.
+
+## Breaking Changes
```

Comment:
~~~suggestion
## How to Update

Run the following command in your project:

```bash npm2yarn
npm install @medusajs/ui@2.0.0
```

---

## Breaking Changes
~~~